app/models/user.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
+    {
+      first_name: DataTypes.STRING,
```

Comment:
In the model the naming should be in camelCase you could use the underscored option and this transform the naming in camelCase to snacke_case. For example 

```
module.exports = (sequelize, DataTypes) => {
  const Buyer = sequelize.define(
    'Buyer',
    {
      id: { allowNull: false, autoIncrement: true, primaryKey: true, type: DataTypes.INTEGER },
      userId: { type: DataTypes.INTEGER, allowNull: false },
      companyId: { type: DataTypes.INTEGER, allowNull: false },
      statusId: { type: DataTypes.INTEGER, allowNull: false },
      createdAt: { allowNull: false, type: DataTypes.DATE },
      updatedAt: { allowNull: false, type: DataTypes.DATE },
      deletedAt: { allowNull: true, type: DataTypes.DATE }
    },
    {
      timestamps: true,
      underscored: true,
      paranoid: true
    }
  );

  Buyer.associate = models => {
    Buyer.belongsTo(models.User, {
      as: 'user',
      foreignKey: 'user_id'
    });
    Buyer.belongsTo(models.Company, {
      as: 'company',
      foreignKey: 'company_id'
    });
    Buyer.belongsTo(models.State, {
      as: 'state',
      foreignKey: 'status_id'
    });
  };

  return Buyer;
};
```